Podfile

```diff
@@ -19,3 +19,13 @@ target 'COVIDWatch iOS' do
   end
 
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    if ['FirebaseFirestoreSwift'].include? target.name
+      target.build_configurations.each do |config|
+        config.build_settings['SWIFT_VERSION'] = '5.0'
```

Comment:
FirebaseFirestoreSwift already builds as swift 5, but the pod spec is set to swift 4.